test/config/utility.cc

```diff
@@ -1109,6 +1109,18 @@ void ConfigHelper::setOutboundFramesLimits(uint32_t max_all_frames, uint32_t max
   }
 }
 
+void ConfigHelper::setUpstreamOutboundFramesLimits(uint32_t max_all_frames,
```

Comment:
Similar to @antoniovicente suggestion, can you please rename `setOutboundFramesLimits` to `setDownstreamOutboundFramesLimits`.

Reply:
done